### PR TITLE
Migrate vivit tutorial to Keras3(all backends)

### PR DESCRIPTION
**Pull request by SuryanarayanaY**
Migrate `'vivit'` tutorial to Keras3(all backends).

Dependency with `tensorflow` for data preprocessing.

Tested with `torch` and `tensorflow` backends and works fine as per [torch-gist](https://colab.sandbox.google.com/gist/SuryanarayanaY/3cf907954e7785eaa69c8c17102f59d9/vivit_git_testing_torch.ipynb#scrollTo=aGkhYDJB2XLQ) and [tf-gist](https://colab.sandbox.google.com/gist/SuryanarayanaY/cdc1c74e1d1946a3d0914cf711be5017/vivit_git_testing_tf.ipynb#scrollTo=5OpXGvPr2R3R).

Please find git diff file below.
[vivit_git-diff.txt](https://github.com/keras-team/keras-io/files/13935980/vivit_git-diff.txt)

### CODE REVIEW

**Comment by sachinprasadhs** [comment]
Closing this, since the tutorial is already merged here https://github.com/keras-team/keras-io/pull/1724. 
Thanks for the PR

**Comment by SuryanarayanaY** [comment]
Hi @sachinprasadhs 

I have tested the latest merged code. The merged code seems not working with torch backend as per attached [torch-gist](https://colab.research.google.com/gist/SuryanarayanaY/c2c13a6f1652469fb62e675a79ae6948/vivit_keras3_contribuor_code-torch.ipynb). Its working with [tf](https://colab.research.google.com/gist/SuryanarayanaY/3803d03fc14a9a72a47fbac0bbb6a073/vivit_keras3_contribuor_code-tf.ipynb) only which is default backend.

Could you please cross check once with torch backend. Thanks!

**Comment by SuryanarayanaY** [comment]
@sachinprasadhs ,

Updated the tutorial with latest merged code and tested with all backends also.
May refer gist for [jax](https://colab.research.google.com/gist/SuryanarayanaY/be8bd6ca1a84ba847d21797a9971f835/vivit_keras3_latest_code_edited-jax-success.ipynb), [tensorflow](https://colab.research.google.com/gist/SuryanarayanaY/9527553a0dc9a06c022d780120c664a2/vivit_keras3_latest_code_edited-tf-success.ipynb) & [torch](https://colab.research.google.com/gist/SuryanarayanaY/0123a87a2d678e491deee30bea58b13c/vivit_keras3_latest_code_edited-torch.ipynb).

**Review by divyashreepathihalli — CHANGES_REQUESTED** [review]
Thank you for the PR. I have left just one comment. Also please generate the .md and .ipynb files.

File: examples/vision/vivit.py [hunk 1]
```diff
@@ -147,8 +148,7 @@ def preprocess(frames: tf.Tensor, label: tf.Tensor):
         ],  # The new axis is to help for further processing with Conv3D layers
         tf.float32,
     )
-    # Parse label
-    label = tf.cast(label, tf.float32)
+    label = tf.cast(label, "float32")
```
[the remark this answers is not in the document]
Comment (SuryanarayanaY) [reply examples/vision/vivit.py:151]: Same problem. Won't work with torch and jax.

File: examples/vision/vivit.py [hunk 2]
```diff
@@ -137,8 +139,7 @@ def download_and_prepare_dataset(data_info: dict):
 """
 
 
-@tf.function
-def preprocess(frames: tf.Tensor, label: tf.Tensor):
+def preprocess(frames, label):
     """Preprocess the frames tensors and parse the labels."""
     # Preprocess images
     frames = tf.image.convert_image_dtype(
```
[the remark this answers is not in the document]
Comment (divyashreepathihalli) [reply examples/vision/vivit.py:145]: Oh okay, if it is used in tf.data, you can leave it as is with tf ops.

**Comment by sachinprasadhs** [comment]
Your branch stills seems to be behind the master branch for the files you modified, could you please make the changes to the updated files to avoid below conflicts.
```
Conflicting files
examples/vision/ipynb/vivit.ipynb
examples/vision/md/vivit.md
examples/vision/vivit.py
```

**Comment by SuryanarayanaY** [comment]
Resolved the conflicts.

**Review by divyashreepathihalli — APPROVED** [review]

**Comment by SuryanarayanaY** [comment]
Fresh PR#1739 done for this due to merge conflicts.

**Pull request closed without merging**